Packs/Exabeam/Integrations/Exabeam/Exabeam.py

```diff
@@ -1816,10 +1819,11 @@ def get_notable_session_details(client: Client, args: Dict[str, str]) -> Tuple[A
         })
 
     contents_entry = {'sessions': contents, 'users': users, 'executiveUserFlags': executive_user_flags}
-
     entry_context = {'Exabeam.NotableSession(val.SessionID && val.SessionID === obj.SessionID)': contents_entry}
-
-    human_readable = tableToMarkdown('Notable Session details:', session, removeNull=True)
+    if sessions and sessions[-1]:
```

Comment:
What do we get in sessions[-1]?

Reply:
The last session, was ordered by user preference.
The issue arose because the `session` var was called out of the loop, which seems like the design was meant to display the last session in the HR.

Reply:
@AradCarmi That sounds right.
Still weird to do so, if we have more than one session, why return just the last one? and not all?
Let's verify it with a TPM